index.js

```diff
@@ -0,0 +1,88 @@
+/* Set Date to Current Date */
+Date.prototype.toDateInputValue = (function() {
+  var local = new Date(this);
+  local.setMinutes(this.getMinutes() - this.getTimezoneOffset());
+  return local.toJSON().slice(0,10);
+});
+
+document.getElementById('implementationDate').value = new Date().toDateInputValue();
+
+/* Contact Form Validation */
+
+const getForm = document.querySelector('.contact-form');
+
+getForm.addEventListener('submit', function(e){
+  e.preventDefault();
+
+  const items = getForm.querySelectorAll('.cf-item');
+  const formItems = getForm.querySelectorAll('.cf-input');
+
+  const resetForm = function() {
+    for (i = 0; i < items.length; i++) {
+      formItems[i].style.borderColor = '#ccc';
+      const errors = items[i].querySelectorAll('.cf-error');
+      for(j = 0; j < errors.length+1; j++) {
+        if(errors[j]) {
+          items[i].removeChild(errors[j]);
```

Comment:
This is running twice because it's a for loop inside a for loop. I'd separate out . the form loops and change the styling for the inputs in one, and remove the error messages in another.

Reply:
An attempt was made. Will discuss directly with @ShooX512